web/app/models.py

```diff
@@ -226,5 +223,30 @@ def __str__(self):
             self.director,
             self.rule_set,
             self.submitted)
+
+class TournamentPlayer(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    name = db.Column(db.String(80))
+    aga_num = db.Column(db.String(5))
+    member_ex_date = db.Column(db.String(20))
+    rating = db.Column(db.String(5))
+    affiliation = db.Column(db.String(80))
+    state = db.Column(db.String(2))
+    address = db.Column(db.String(80))
+    email = db.Column(db.String(80))
+    phone = db.Column(db.String(20))
+    citizenship = db.Column(db.String(80))
+    dob = db.Column(db.String(20))
+    tournament_id = db.Column(db.Integer, db.ForeignKey('tournament.id'))
+    tournament = relationship(Tournament,
+        backref=db.backref('Tournament', cascade='all, delete-orphan'))
+
+
+    def __str__(self):
+        return "Tournament Player: {}\n \
+                Member found in db: {}".format(self.name,
+                                                player_id >= 90000)
```

Comment:
embarrassingly, I can't remember what I was thinking there. I think it was a half-formed thought about returning a message if the tournament player was found in our database but... half-formed thought seems generous right now. I'll clean that up.